calendar-bundle/src/Picker/EventPickerProvider.php

```diff
@@ -80,7 +80,7 @@ public function convertDcaValue(PickerConfig $config, $value): string
         return sprintf($this->getInsertTag($config), $value);
     }
 
-    protected function getRouteParameters(PickerConfig $config = null): array
+    protected function getRouteParameters(?PickerConfig $config = null): array
```

Comment:
```suggestion
    protected function getRouteParameters(PickerConfig|null $config = null): array
```
We use an explicit `|null` in our types instead of the `?` prefix AFAIK.

Reply:
Contao 4.13 supports PHP 7.4 still, that's why